bin/dartdoc.dart

```diff
@@ -54,6 +56,16 @@ void main(List<String> arguments) {
     ..generateDocs();
 }
 
+exitWhenRunFromSnapshot() {
+  var script = Platform.script;
+  print(script);
+  if (script.toString().contains('snapshot')) {
```

Comment:
The intention of this PR changed. We will support running in a globally activated snapshot. So I'll remove this :)